pkgs/development/tools/supabase-cli/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, buildGo118Module
```

```suggestion
, buildGoModule
```

Why do we need Go 1.18? If this is really required this must be commented.

---

works with 1.19, must have misread the docs of supabase